src/index.html

```diff
@@ -1,13 +1,107 @@
+<!-- #region start -->
 <!doctype html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
+
     <title>Product cards</title>
-    <link rel="stylesheet" href="./styles/index.scss">
+
+    <link
+      rel="stylesheet"
+      href="./styles/index.scss"
+    >
```

Comment:
You should move it to the new line if  the HTML-element has long attribute values or number of attributes is more than 2
```suggestion
    <link rel="stylesheet" href="./styles/index.scss">
```